### DISCUSSION TITLE
Custom allies

### DISCUSSION THREAD

**Discussion by suurimonster** [Lurkars/gloomhavensecretariat#313] [Q&A] [answered]
In certain situations, it is possible to have an ally that is not connected to the scenario. I could not see an obvious way to handle this and we basically just did it manually (health counter and ally deck). Is this not possible in the current version? I accept it is a pretty rare event and not a priority.

**Comment by Lurkars** [comment, marked as answer]
Totally possible, just bad UX:

- Click on the  <img src="https://gloomhaven-secretariat.de/assets/images/monster/enemy.png" width="20" alt="Scull Icon"/> Icon on the top right of the monster stat sheet to toggle `Ally` status.

For reference: double click (long press on mobile) on the icon, to toggle `Allied` status, which differs, that all `Allied` monsters are allies to another but enemies for all others (so not an `Ally` for characters).
